translations/strings.yaml

```diff
@@ -683,13 +684,12 @@ global:
   collapse:
     value: Collapse
     comment: Action used in the context of an expandable/collapsible card component
-  vpnUser: 
+  vpnUser:
     value: VPN User
     comment: Fallback string for users who have not specified a Display Name.
-  discoverNow: 
+  discoverNow:
     value: Discover now
     comment: Label for button used in intro modals to discover new features.
   goBack:
     value: Go back
     comment: Label for button used in intro modals to return to the app.
```

Comment:
The other changes are nice (there should be no trailing spaces in this file, and I'm all for consistency), but please keep an empty line at the end of this file (I thought we added checks for it, but I might be wrong).

Reply:
Hum, new lines at the end of files is actually I something my VSCode _should_ be setup to add on save 🤔  Anyways, I'll re-add it.

Reply:
Might something related to a package for YAML? I don't think VS Code reformats YAML out of the box.

Reply:
Hum, it's the prettier formatter. It now supports not only JavaScript but also Markdown and YAML apparently. That explains all these reformattings.

Anyways, so this is fun. Maybe we found a bug in git? So here is what is happening: there _is_ actually a new line in my local file, VSCode always adds a new line when I save and trims extra new lines. I think git is confused, because there were trailing spaces there which were removed. See:

<img width="828" alt="Screenshot 2022-07-21 at 17 15 05" src="https://user-images.githubusercontent.com/25176023/180250477-32467853-0eec-445a-8776-5e088196114d.png">

I sent a commit adding the newline: https://github.com/mozilla-mobile/mozilla-vpn-client/pull/4046/commits/e28d0fc58ebf1a6b420feac1b0d2e82e75d19159, and now git is happy.

However in order to do that I had to disable my formatter on VSCode, because that will strip extra new lines... And probably will have to remember to do that always when editing this file, because even though it now looks corerct for git, this is how it looks for me:

<img width="748" alt="Screenshot 2022-07-21 at 17 34 21" src="https://user-images.githubusercontent.com/25176023/180254142-25495b34-65d9-4efc-b02c-7f7152a759ab.png">

YAY!